src/Pingalot.Core/PingSession.cs

```diff
@@ -33,7 +33,7 @@ public PingSession(DateTime startTime, DateTime endTime, TimeSpan elapsed, IRead
 		private void CalculateStatistics()
 		{
 			PacketsSent = Requests.Count;
-			var totalRoundtrip = 0L;
+			double totalRoundtrip = 0;
```

Comment:
We can actually just replace the `L` with a `d` to make it a `double` without explicitly setting the type.